platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/Migratable.java

```diff
@@ -46,13 +46,4 @@
      */
     @NotNull
     String getDescription();
```

Comment:
We need to update the interface's JavaDoc to make it clear that no one should implement this interface but should instead implement `DataMigratable` or `ConfigurationMigratable`.

It also needs to extend from `Describable` when that new common interface has been merged.